tango_simlib/tests/test_tango_sim_generator.py

```diff
@@ -154,3 +162,122 @@ def test_sim_control_device_attribute_change(self):
         self.sim_control_device.pause_active = True
         setattr(self.sim_control_device, 'last_val', input_value)
         self.assertEqual(self.sim_device.temperature, input_value)
+
+class test_TangoSimGenDeviceIntegrationForFandangoFile(ClassCleanupUnittestMixin,
+    unittest.TestCase):
```

Comment:
- [ ] Don't think the indentation here isn't quite pep8 compliant.  Here are some options:
```
class test_TangoSimGenDeviceIntegrationForFandangoFile(ClassCleanupUnittestMixin,
                                                       unittest.TestCase)
```
or
```
class test_TangoSimGenDeviceIntegrationForFandangoFile(
    ClassCleanupUnittestMixin, unittest.TestCase)
```